content/en/tracing/profiler/enabling/php.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Enabling the PHP Profiler
+kind: Documentation
+code_lang: php
+type: multi-code-lang
+code_lang_weight: 50
+further_reading:
+    - link: 'getting_started/profiler'
+      tag: 'Documentation'
+      text: 'Getting Started with Profiler'
+    - link: 'tracing/profiler/search_profiles'
+      tag: 'Documentation'
+      text: 'Learn more about available profile types'
+    - link: 'tracing/profiler/profiler_troubleshooting'
+      tag: 'Documentation'
+      text: 'Fix problems you encounter while using the profiler'
+---
+
+{{< site-region region="us5" >}}
+<div class="alert alert-warning">
+  The Continuous Profiler is not available for the Datadog {{< region-param key="dd_site_name" >}} site.
+</div>
+{{< /site-region >}}
+
+<div class="alert alert-warning">
+The Datadog PHP Profiler is currently in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
+</div>
+
+Starting in tracer version 0.69.0, both the tracer and profiler can be installed simultaneously using the script `datadog-setup.php`.
+
+## Requirements
+
+The Datadog Profiler requires at least PHP 7.1. PHP 8.1 is not yet supported. ZTS builds of PHP are not supported, nor are debug builds.
+
+A 64-bit, Linux Operating System (OS) is required.
+
+{{< tabs >}}
+{{% tab "GNU C Linux" %}}
+
+An Operating System with glibc 2.17 or newer is needed. The following OS versions or newer meet this requirement:
+  - CentOS 7
+  - Debian 8, which has reached End Of Life (EOL).
+  - Ubuntu 14.04, which is EOL.
+
+Datadog recommends running a supported OS version.
+
+The following libraries must be installed:
+  - `libc.so.6`
+  - `libpthread.so.0`
+  - `libdl.so.2`
+  - `libm.so.6`
+
+The tracer also requires these libraries:
+  - `libcurl.so.4`
+  - `librt.so.1`
+
+{{% /tab %}}
+{{% tab "Alpine Linux" %}}
+
+Version 3.13 of Alpine Linux or newer is required. The following libraries must be installed:
+  - `libc.musl-x86_64.so.1`
+
+The tracer also requires these libraries:
+  - `libcurl.so.4`
+  - `libexecinfo.so.1`
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Installation
+
+To begin profiling applications:
+
+ 1. If you are already using Datadog, upgrade your Agent to version [7.20.2][2]+ or [6.20.2][3]+.
+
+ 2. Download the `datadog-setup.php` script from the [GitHub release page](https://github.com/DataDog/dd-trace-php/releases). Version 0.69.0 was the first release to include this installer. Verify its sha256sum with the values provided on the release page.
```

Comment:
We don't say to verify the checksum for other languages. We should be consistent one way or the other.

IMO we should not include this. The checksums are there for users who want to check them. For the others, we really want to keep things as dead simple as we can.